proposals/00-00-template.md

```diff
@@ -0,0 +1,159 @@
+<!--
+**Note:** When your proposal is complete, all of these comment blocks should be removed.
+
+To get started with this template:
+
+- [ ] **Make a copy of this file.**
+  Name it `YY-MM-short-descriptive-title.md` (where `YY-MM` is the current year and month).
+- [ ] **Fill out this file as best you can.**
+  At minimum, you should fill in the "Summary" and "Motivation" sections.
+- [ ] **Create a PR.**
+  Ping `@kubernetes-sigs/kustomize-admins` and `@kubernetes-sigs/kustomize-maintainers`.
+-->
+
+# Your short, descriptive title
+
+**Authors**:
+
+**Reviewers**:
```

Comment:
Is the reviewers same as approvers? Can we give some doc to mention who can be a qualified reviewer (like kustomize members)?

Reply:
Here's the doc on the Kustomize contributor ladder: https://github.com/kubernetes-sigs/kustomize/blob/master/CONTRIBUTING.md#contributor-ladder. I'd say anyone on either list (admin or maintainer) can be a reviewer here.

Reply:
Gotcha. Thanks!